corehq/apps/app_manager/views/apps.py

```diff
@@ -460,10 +460,9 @@ def _copy_app_helper(request, master_domain, master_app_id_or_source, copy_to_do
     try:
         app_copy = import_app_util(master_app_id_or_source, copy_to_domain, extra_properties)
     except ReportConfigurationNotFoundError:
```

Comment:
Does whatever raises this exception not short-circuit the rest of the copying process?

Reply:
yeah, this is confusing...

Reply:
The copying process gets as far as:
https://github.com/dimagi/commcare-hq/blob/ab43130d709daf8ae3129fb985f6181ad7c55dd9/corehq/apps/app_manager/models.py#L5858

The only thing that would be broken is multimedia.  Based on this previous ticket:
https://dimagi-dev.atlassian.net/browse/ICDS-240
it's only advanced Dimagi users that will ever run into this issue.  Probably fine if a warning about multimedia is added as well.

Reply:
Updated: e85e2a1ccf8

Reply:
This still feels like an odd place for this check - could this check be moved to the `import_app` function, perhaps?  That way it's caught where it's expected to be raised and it's clear what's being skipped.  I understand that'd make this message harder to display, maybe pass back a list of "warnings" that can be shown to the users or something.  IDK, that's not a perfect solution either.

Reply:
Here's an alternative approach: 3a430de54a4